src/pkg/storage/testdata/storage.go

```diff
@@ -51,6 +52,28 @@ func NewPrefixedS3Storage(t tester.TestT) storage.Storage {
 	return st
 }
 
+func NewFilesystemStorage(t tester.TestT) storage.Storage {
+	now := tester.LogTimeOfTest(t)
+	repoPath := filepath.Join(t.TempDir(), now)
+
+	err := os.MkdirAll(repoPath, 0700)
+	require.NoError(t, err, "creating filesystem repo", clues.ToCore(err))
+
+	t.Logf("testing at filesystem repo [%s]", repoPath)
+
+	st, err := storage.NewStorage(
+		storage.ProviderFilesystem,
+		&storage.FilesystemConfig{
+			Path: repoPath,
+		},
+		storage.CommonConfig{
+			Corso: GetAndInsertCorso(""),
+		})
+	require.NoError(t, err, "creating storage", clues.ToCore(err))
```

Comment:
Formatting is required if the assertion contains a message.
```suggestion
	require.NoErrorf(t, err, "creating storage: %+v", clues.ToCore(err))
```